README.md

```diff
@@ -504,6 +504,12 @@ Jimp.rgbaToInt(r, g, b, a); // e.g. converts 255, 255, 255, 255 to 0xFFFFFFFF
 Jimp.intToRGBA(hex); // e.g. converts 0xFFFFFFFF to {r: 255, g: 255, b: 255, a:255}
 ```
 
+You can convert a css hex color to its true rgba hexadecimal equivalent:
```

Comment:
It seems like tinycolor2 can take more than just a hex value. 

https://www.npmjs.com/package/tinycolor2

Can you update this to reflect that you can also pass in RGB, RGBA, HSL, HSLA, HSV, HSVA, and named values also?